### ISSUE TITLE
Pasted multiple lines read as one line

### ISSUE THREAD

**Issue by harmic** [kkawakam/rustyline#202]
I found one strange thing when running the example.

I am running on a Linux machine, and connected via an ssh session from a windows desktop using putty. Entering text normally works as expected.

However if I have a number of lines in an editor, and copy/paste those into the terminal, somehow they are all considered a single multi-line string (with embedded newlines) rather than multiple lines. Only pressing the enter key on the keyboard terminates the line.

```
 $ cargo run --example example
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/examples/example`
>> 1
2
3
4


Line: 1
2
3
4

>>

```

At first I thought some kind of line ending difference was causing this, but pasting into a file looks exactly the same as entering newlines manually - they are all seen as 0x0a (\n).

It seems rustyline somehow treats pasted \n characters differently to newline pressed on the keyboard. This doesn't seem to be the case when using other readline implementations.

**Comment by gwenn** [comment]
This is related to #194.
See [bracketed paste mode](https://cirw.in/blog/bracketed-paste).

**Comment by erikgrinaker** [comment]
For anyone wondering, this behavior can be disabled like this:

```rust
editor.bind_sequence(rustyline::KeyPress::BracketedPasteStart, rustyline::Cmd::Noop);
```

Thanks for making Rustyline, love it! ❤️

**Comment by gwenn** [comment]
Fixed in version 8.0.0
